bin/11_16/src/Main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <algorithm>
+#include <exception>
+#include <fstream>
+#include <iostream>
+#include <iterator>
+#include <utility>
+#include <vector>
+
+using Pair = std::pair<int, int>;
+
+void Print(const std::vector<Pair>& values) {
+  for (const auto value : values) {
+    std::cout << value.first;
+    if (value.second > 1) {
+      std::cout << " " << value.second;
+    }
+    std::cout << std::endl;
+  }
+}
+
+std::vector<Pair> Repack(const std::vector<int>& values) {
+  std::vector<Pair> resorted_values;
+  for (const auto value : values) {
+    resorted_values.push_back({
+      value,
+      std::count(values.begin(), values.end(), value)
+    });
+  }
+  resorted_values.erase(
+    std::unique(
+      resorted_values.begin(),
+      resorted_values.end()),
+    resorted_values.end());
+  return resorted_values;
+}
```

Comment:
This is not an optimal solution, I suggest you use std::equal_range algorithm to find all subsequences of equal elements.